ci-operator/step-registry/confidential-clusters/ref/cocl/e2e-test/confidential-clusters-ref-cocl-e2e-test-commands.sh

```diff
@@ -0,0 +1,1040 @@
+#!/bin/bash
```

Comment:
I don' think this script is needed, we used the fcos image from investigation for our local testing, but this isn't necessary for e2e tests